### PR TITLE
create and use palantir-hadoop profile

### PR DESCRIPTION
**Pull request by sjrand**
@robert3005 @pwoody @ash211

### CODE REVIEW

**Comment by sjrand** [comment]
Apparently the build failed because of `bash: line 7: BINTRAY_USERNAME: unbound variable`. Does this just mean that my fork doesn't have access to that env variable?

File: pom.xml [hunk 1]
```diff
@@ -2543,6 +2543,16 @@
     </profile>
 
     <profile>
+      <id>palantir-hadoop</id>
```
Comment (ash211) [inline pom.xml]: call this `hadoop-palantir` to match the other profiles `hadoop-X` format

**Comment by ash211** [comment]
@sjrand yes I believe that env variables are only available for forks coming from the github.com/palantir repo, not personal repos.  Change your PR to be from `palantir/spark` and I think that'll work

**Comment by sjrand** [comment]
I don't think I can make the PR from `palantir/spark` without write access to the repo?

**Comment by sjrand** [comment]
(In other words can I have write perms?)

**Comment by ash211** [comment]
@sjrand added

**Comment by sjrand** [comment]
Closed in favor of #54, which is the same change, but from a branch within this repo.

**Pull request closed without merging**